guacamole_sg.tf

```
@@ -42,13 +41,13 @@ resource "aws_security_group_rule" "guacamole_egress_to_hosts_via_vnc" {
 resource "aws_security_group_rule" "guacamole_ingress_from_trusted_via_https" {
   provider = aws.provisionassessment
 
+  cidr_blocks = [local.vpn_server_cidr_block]
+  # ipv6_cidr_blocks  = TBD
+  from_port         = 443
+  protocol          = "tcp"
```

In the event that we ever uncomment this line, it will be in the correct alphabetical order:

```suggestion
  from_port         = 443
  # ipv6_cidr_blocks  = TBD
  protocol          = "tcp"
```